packages/config/src/get-mod-deps.ts

```diff
@@ -97,7 +97,7 @@ function getDeps(opts: GetDepsOptions, fromAlias?: string) {
       // this is for internal monorepo packages that don't have a `dist`
       // and instead use a package.json `main` field that points to a src/xxx.ts file
       const found = ts.resolveModuleName(mod, absoluteFile, compilerOptions, ts.sys).resolvedModule
-      if (found && found.extension === '.ts') {
+      if (found && found.extension.endsWith('.ts')) {
```

Comment:
Should this be `ts` without the `.`? This wouldn't pick up `.mts` or `.d.mts` as mentioned in the context 
```suggestion
      if (found && found.extension.endsWith('ts')) {
```

Reply:
you're completely right !